task/06-conditions-n-loops-tasks.js

```diff
@@ -270,7 +350,25 @@ function isCreditCardNumber(ccn) {
  *   165536 (1+6+5+5+3+6 = 26,  2+6 = 8) => 8
  */
 function getDigitalRoot(num) {
-    throw new Error('Not implemented');
+    let result = countSumOfDigitals(num);
```

Comment:
Here you can return the result of a function without using variable or even do without a nested function.

Reply:
Fixed.